content/en/docs/guides/flux-e2e.md

```diff
@@ -0,0 +1,399 @@
+---
+title: "Flux from End-to-End"
+linkTitle: "Flux from End-to-End"
+description: "A narrative of the life of a commit as it relates to Flux components."
+weight: 75
+---
+
+Below we describe the flow of data through Flux, from End to End.
+
+We assume a standard Flux installation, with all optional features enabled, then explain how Flux users can expect their changes to flow through the stages of its life as a commit, and describe how the commit data passes through the Flux system and the cluster, in rough chronological order. We tried to cover every supported opportunity that users have to inspect and interact with their changes through Flux, with a goal of showing the cohesive behavior of each component of the [GitOps toolkit][] together in a single document.
```

Comment:
Maybe link to the instructions for a standard Flux installation? Is it just `/docs/get-started`?

Reply:
Added in efe433a 👍